browserbase/helpers/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .gpt4 import GPT4VImage, GPT4VImageDetail
+from .anthropic import Claude3Image
+
+__all__ = ["GPT4VImage", "GPT4VImageDetail", "Claude3Image"]
```

Comment:
seems like a no-op; wouldn't a user theoretically just import `Claude3Image` directly?

Reply:
![image](https://github.com/user-attachments/assets/22b3072b-f9fe-44cb-ba58-61db16138207)

some context, langchain was using our sdk and the imports weren't correct.

https://docs.browserbase.com/integrations/langchain/python
If u ctrl f gpt4

![image](https://github.com/user-attachments/assets/cbfa2ade-c4c0-44ea-850c-f61e1a0c6f6d)

Reply:
ah i see yeah sg

Reply:
also fwiw it looks like you're importing from `browserbase.helpers.gpt4` instead of `browserbase.helpers`

Reply:
the problem is that the code was copy and pasted into the docs for both browserbase and langchain, so should we work around that?

Reply:
Can we just edit their page to make it correct? https://python.langchain.com/docs/integrations/document_loaders/browserbase/#loading-images (link at bottom)

Reply:
![CleanShot 2024-10-01 at 16 45 11@2x](https://github.com/user-attachments/assets/3300894d-887c-4c5e-8159-a0503e13a2c4)

still getting this when using `browserbase.helpers`